app/Console/Kernel.php

```diff
@@ -32,6 +32,6 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        //
+        $schedule->command('queue:work --sleep=3 --tries=3')->everyMinute();
```

Comment:
I'd say we should just leave people with nothing forced out of the box like laravel does. People can setup something like this if they want it imo.